esp-link/config.c

```diff
@@ -9,21 +9,31 @@
 
 FlashConfig flashConfig;
 FlashConfig flashDefault = {
-  33, 0, 0,
-  MCU_RESET_PIN, MCU_ISP_PIN, LED_CONN_PIN, LED_SERIAL_PIN,
-  115200,
-  "esp-link\0",                        // hostname
-  0, 0x00ffffff, 0,                    // static ip, netmask, gateway
-  0,                                   // log mode
-  0,                                   // swap uart (don't by default)
-  1, 0,                                // tcp_enable, rssi_enable
-  "\0",                                // api_key
-  0, 0, 0,                             // slip_enable, mqtt_enable, mqtt_status_enable
-  2, 1,                                // mqtt_timeout, mqtt_clean_session
-  1883, 60,                            // mqtt port, mqtt_keepalive
-  "\0", "\0", "\0", "\0", "\0",        // mqtt host, client_id, user, password, status-topic
-  "\0",                                // system description
-  1,                                   // rx_pullup
+  .seq=33, .magic=0, .crc=0,
+  .reset_pin    = MCU_RESET_PIN, .isp_pin      = MCU_ISP_PIN,
+  .conn_led_pin = LED_CONN_PIN,  .ser_led_pin  = LED_SERIAL_PIN,
+  .baud_rate    = 115200,
+  .hostname     = "ems-link02\0",
+  .staticip     = 0,
+  .netmask      = 0x00ffffff,
+  .gateway      = 0,
+  .log_mode     = 0,
+  .swap_uart    = 0,
+  .tcp_enable   = 1, .rssi_enable = 0,
+  .api_key      = "\0",
+  .slip_enable  = 0, .mqtt_enable = 0, .mqtt_status_enable = 0,
+  .mqtt_timeout = 2, .mqtt_clean_session = 1,
+  .mqtt_port    = 1883, .mqtt_keepalive = 60,
+  .mqtt_host    = "\0", .mqtt_clientid  = "\0",
+  .mqtt_username= "\0", .mqtt_password = "\0", .mqtt_status_topic = "\0",
+  .sys_descr 	= "\0",
+  .rx_pullup	= 1,
+  .sntp_server  = "diskstation\0",
+  .syslog_host  = "diskstation\0",
+  .syslog_minheap= 8192,
```

Comment:
It's not the size of the syslog buffer - it's the required overall heapsize when (trying to) queue a syslog message.